### DISCUSSION TITLE
Warehouse management and features

### DISCUSSION THREAD

**Discussion by raghav4882** [lovata/oc-shopaholic-plugin#361] [Ideas]
So I have been looking into this plugin for a while now and the most important feature that I have been looking for is multi-point proper warehouse management. for example, if my company has two warehouses in one country then estimated dates and points of sending should be from the closest warehousing team. (if warehouse in new jersey and Los Angeles, then a purchase made from New York will automatically be assigned to the nearest one and stock will be subtracted from that).

Is there anything that can help me achieve this? Also, if I want to implement this multiple warehouse model in multiple countries; how can one go about that?
Thanks for your time reading this.

**Comment by kharanenka** [comment]
Hi! We do not yet have full support for warehouses. Now, for a simple catalog, a catalog structure may be suitable, when offers are product leftovers in different warehouses. But this is not correct logic in the future.

We have plans to add warehouse functions.

Now you can add new model for saving offer quantity and link it with Offer model. Perhaps this will be an easy solution for you now.

**Comment by lautsevich** [comment]
@raghav4882, hi! As @kharanenka said this feature is on our roadmap but with the low priority. There's an option to boost via donation this feature development. If you're interested, please, drop a line at shopaholic@shopaholic.one.
